### ISSUE TITLE
Empty category - Cannot modify header information

### ISSUE THREAD

**Issue by barrycrous-zz** [SimonPadbury/b4st#54]
Hi Simon,

Great job on this theme. We just picked up one thing, when viewing an empty category, this error pops up:

Warning: Cannot modify header information - headers already sent by (output started at D:\Xampp\htdocs\triomfsa\wp-includes\class.wp-scripts.php:343) in D:\Xampp\htdocs\triomfsa\wp-includes\pluggable.php on line 1216

Cheers

**Comment by SimonPadbury** [comment]
Hi Barry,

Thankyou for this. I replicated the error. And by process of elimination (commenting out), I now only wee the error when  **any of the first 4 register+enqueues** in `functions/enqueues.php` are **not** commented out.

And, I swapped those 4 for the older versions of the same -- but I got the same error message.So this is a longstanding problem that has been missed.

Sorry I can't work on this further now, I will drop on to it again later.

Cheers!
